tm2/pkg/sdk/params/handler.go

```diff
@@ -28,14 +28,13 @@ func (bh paramsHandler) Process(ctx sdk.Context, msg std.Msg) sdk.Result {
 // Query
 
 func (bh paramsHandler) Query(ctx sdk.Context, req abci.RequestQuery) (res abci.ResponseQuery) {
-	switch secondPart(req.Path) {
-	case bh.params.prefix:
+	prefix := secondPart(req.Path)
```

Comment:
kinda prefer the switch because it sets a simple convention and theoretically takes advantage of switch-on-string optimizations.